cf-agent/package_module.c

```diff
@@ -1125,9 +1125,9 @@ PromiseResult FileInstallPackage(const char *package_file_path,
      
     if (action == cfa_warn || DONTDO)
     {
-         Log(LOG_LEVEL_WARNING, "Should install file type package: %s",
+         Log(LOG_LEVEL_VERBOSE, "Should install file type package: %s",
```

Comment:
Yes, there is no need for two messages with almost the same content at warn level.